schemas/compat-data.schema.json

```diff
@@ -35,25 +35,38 @@
               }
             },
             "additionalProperties": false,
-            "required": ["type", "name"]
+            "required": ["type", "name"],
+            "minProperties": 0,
+            "maxProperties": 3
           }
         },
         "partial_implementation": {
-          "type": "boolean",
+          "const": true,
```

Comment:
This is the only change that's a little stricter than what we already allow, but technically speaking we don't really allow any values for `partial_implementation` other than `true` if it's defined anyways.

Reply:
Does the lint enforce this? If so I think this change is fine.

Reply:
Surprisingly it isn't enforced by the linter, but just as surprising is that we've not had a value other than `true` show up in our data.  This will enforce it in a linter though!